imports/client/components/HuntApp.tsx

```diff
@@ -144,7 +155,29 @@ const HuntApp = React.memo(() => {
     return <HuntMemberError hunt={hunt} canJoin={canJoin} />;
   }
 
-  return <Outlet />;
+  let termsModal = null;
+  if (mustAcceptTerms) {
+    termsModal = createPortal(
+      <Modal show size="lg">
+        <ModalBody>
+          <Markdown text={hunt.termsOfUse!} />
+        </ModalBody>
+        <ModalFooter>
+          <Button variant="primary" onClick={acceptTerms}>
```

Comment:
How annoyed do you think people would be if we made this button stay disabled for a few seconds after being shown, before enabling, to encourage people to actually take the time to read the reminders?

Reply:
Ehhh, I changed my mind, that's probably overkill.

Reply:
Not necessarily opposed but yeah seems like it's probably overkill